src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/config/OpenApiConfiguration.kt

```diff
@@ -0,0 +1,110 @@
+package uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.config
+
+import io.swagger.v3.oas.models.Components
+import io.swagger.v3.oas.models.OpenAPI
+import io.swagger.v3.oas.models.Operation
+import io.swagger.v3.oas.models.info.Contact
+import io.swagger.v3.oas.models.info.Info
+import io.swagger.v3.oas.models.security.SecurityRequirement
+import io.swagger.v3.oas.models.security.SecurityScheme
+import org.springdoc.core.customizers.OperationCustomizer
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.info.BuildProperties
+import org.springframework.context.ApplicationContext
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.expression.BeanFactoryResolver
+import org.springframework.expression.spel.SpelEvaluationException
+import org.springframework.expression.spel.standard.SpelExpressionParser
+import org.springframework.expression.spel.support.StandardEvaluationContext
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.method.HandlerMethod
+
+@Configuration
+class OpenApiConfiguration(buildProperties: BuildProperties) {
+  private val version: String = buildProperties.version
+
+  @Autowired
+  private lateinit var context: ApplicationContext
+
+  @Bean
+  fun customOpenAPI(): OpenAPI? = OpenAPI()
+    .info(
+      Info()
+        .title("Challenge Support Intervention Plan API")
+        .version(version)
+        .description(
+          "API for retrieving and managing Challenge Support Intervention Plan (CSIP) relating to a person.\n\n" +
```

Comment:
Change to "API for retrieving and managing Challenge, Support and Intervention Plan (CSIP) records relating to a person